src/extension.ts

```diff
@@ -41,7 +41,9 @@ export function activate(context: ExtensionContext) {
     }
   });
   workspace.onDidCloseTextDocument(() => {
-    statusBar.hide();
+    if (!window.activeTextEditor) {

```

Comment:
What is this doing? `activeTextEditor` and `statusBar` seem unrelated.

Reply:
It hides `statusBar` only when there is no active editor. 
I had cases where `onDidClose` came after a `onDidOpen` / `onDidChange`, so `statusBar` was hidden even though the then active editor had some coverage.